### DISCUSSION TITLE
SigmaOS web extension support

### DISCUSSION THREAD

**Discussion by BeltranMal** [pieces-app/support#412] [Ideas]
I believe that having a web extension for SigmaOS is a great idea because pieces and SigmaOS complement each other in an amazing way, a web browser focused in enhancing productivity like sigmaOS isn't really complete without having the ability to complement with pieces snippets and chats.

**Comment by rosie-at-pieces** [comment]
hi @BeltranMal great idea! please add this to this discussion here so we can have all extension requests in one place:

https://github.com/pieces-app/support/discussions/185
